runtime/vm/createramclass.cpp

```diff
@@ -1777,6 +1786,9 @@ loadFlattenableFieldValueClasses(J9VMThread *currentThread, J9ClassLoader *class
 		}
 		field = romFieldsNextDo(&fieldWalkState);
 	}
+	if (eligibleForFastSubstitutability) {
```

Comment:
add a comment here explaining what the criteria is for `J9ClassCanSupportFastSubstitutability`

Reply:
I've added the comment to the javadoc for the method for more clarity since this function achieves multiple purposes